CHANGELOG.md

```diff
@@ -74,7 +74,8 @@ The format is based on [Keep a Changelog].
 * Multiline candidates are now merged into a single truncated line so
   there is no gradual scrolling effect anymore when going through the
   candidate list. The first matched line is shown in front of the
-  merged lines ([#133]).
+  merged lines ([#133]). The indicators used to represent this
```

Comment:
We should mention the new user option here and maybe we could rephrase it a bit? Something like: The formatting is customizable via \`selectrum-multiline-display-settings\`.